head_rpn/model.py

```diff
@@ -34,3 +34,28 @@ def get_regularized_model(config):
     rpn_class = Conv2D(config['anchor_count'], (1, 1), activation='sigmoid', name='RPN_classifier')(output)
     rpn_regr = Conv2D(config['anchor_count'] * 4, (1, 1), activation='linear', name='RPN_regressor')(output)
     return Model(inputs=base_model.input, outputs=[rpn_class, rpn_regr])
+def get_model_for_tuning(config):
+     """
+        Creates the RPN model from the config using L2 regularization
+        Args:
+            config,  the configuration dictionary
+        Returns:
+            the Keras RPN model
+    """
+    image_height, image_width = config['image_height'], config['image_width']
+    base_model = VGG16(include_top=False, input_shape=(image_height, image_width, 3))
+    base_model.trainable = True
+    set_trainable = False
+    for layer in base_model.layers:
+        if layer.name == 'block5_conv1' or layer.name == 'block5_conv2' or layer.name == 'block5_conv3':
+           set_trainable = True
+        if set_trainable:
```

Comment:
This if is useless as for the above comment

Reply:
These lines seems not to lock layers that has different location from block5, following commit contain updates